src/app/modules/project/details/details.component.ts

```diff
@@ -313,24 +311,17 @@ export class DetailsComponent implements OnInit {
    * Method to display the edit project button based on the current user and the project user.
    * If the user either has the ProjectWrite scope or is the creator of the project
    */
-  private determineDisplayEditProjectButton(): void {
+  private determineDisplayEditAndDeleteProjectButton(): void {
     if (this.currentUser == null || this.project == null || this.project.user == null) {
       this.displayEditButton = false;
-      return;
-    }
-    this.displayEditButton = this.project.user.id === this.currentUser.id;
-  }
-
-  /**
-   * Method to display the delete project button based on the current user and the project user.
-   * If the user either has the ProjectWrite scope or is the creator of the project
-   */
-  private determineDisplayDeleteProjectButton(): void {
-    if (this.currentUser == null || this.project == null || this.project.user == null) {
       this.displayDeleteProjectButton = false;
       return;
     }
-    this.displayDeleteProjectButton = this.project.user.id === this.currentUser.id;
+    if (this.project.user.id === this.currentUser.id ||
+        this.authService.getCurrentBackendUser().role.name === 'Administrator') {
```

Comment:
I don't like checking on role name because new roles can get added that have the same permission to access this in the future. Let's change this line to something like this:

```suggestion
        this.authService.currentBackendUserHasScope(scopes.AdminProjectWrite)
```
(This requires adding this new scope from the backend to the enum scopes in the frontend though)